pandas/tests/series/test_api.py

```diff
@@ -497,7 +497,7 @@ def test_empty_method(self):
         pytest.importorskip("IPython", minversion="6.0.0")
         from IPython.core.completer import provisionalcompleter
 
-        code = "import pandas as pd; s = pd.Series()"
+        code = "import pandas as pd; s = pd.Series(dtype=object)"
```

Comment:
This is just silencing a warning I bumped into

Reply:
You should use the context manager to catching FutureWarning instead of doing this